exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -159,7 +171,11 @@ export async function handler (
       : undefined
     return printProjectCommands(manifest, rootManifest ?? undefined)
   }
-  if (scriptName !== 'start' && !manifest.scripts?.[scriptName]) {
+
+  const multiScriptSelectorRegExp = buildRegExpFromCommand(scriptName)
+  const specifiedScripts = multiScriptSelectorRegExp ? Object.keys(manifest.scripts ?? {}).filter(script => script.match(multiScriptSelectorRegExp)) : (!!manifest.scripts?.[scriptName] || scriptName === 'start') ? [scriptName] : []
+
+  if (scriptName !== 'start' && specifiedScripts.length < 1) {
```

Comment:
I don't think scriptName should be used in this function from here to the end. As scriptName may be a regex. Just use the new "specifiedScripts" variable.

Reply:
As you pointed, `specifiedScripts` needs to include the logic which uses scriptName. Thanks !! c1c4cd9

(I changed also the help message a bit according to this PR)